static/app/components/group/tagFacets/tagFacetsDistributionMeter.tsx

```diff
@@ -87,7 +86,7 @@ function TagFacetsDistributionMeter({
       <Title>
         <TitleType>{title}</TitleType>
         <TitleDescription>{topSegments[0].name || t('n/a')}</TitleDescription>
-        {multiValueTag && (
+        {
```

Comment:
Do you still actually need these braces tho?

Reply:
👍 good catch, definitely not needed! Updated!